src/themes/shared/components/Table/tableRow-styles.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  ComponentSlotStylesPrepared,
+  TeamsThemeStylesProps,
+} from "@fluentui/react-northstar";
+
+export const tableRowStyles: ComponentSlotStylesPrepared<
+  NonNullable<Partial<TeamsThemeStylesProps["TableRow"]>>,
+  any
+> = {
+  root: ({
+    props: { header },
+    theme: {
+      siteVariables: { colorScheme },
+    },
+  }) => {
+    return {
+      borderBottomColor: colorScheme.default.border2,
+      ...(header && { "--table-cell__content--font-size": "0.75rem" }),
+      "&:hover": {
+        backgroundColor: `${colorScheme.default.background4} !important`,
+        borderLeftColor: `${colorScheme.default.background4} !important`,
+        borderTopColor: `${colorScheme.default.background4} !important`,
+        borderRightColor: `${colorScheme.default.background4} !important`,
+        borderBottomColor: `${colorScheme.default.border2} !important`,
```

Comment:
Unfortunately the only way to override pseudoclass styles applied by Fluent UI's styling engine (that I've discovered) is to add `!important`. There are other overrides in the HVCs that rely on this technique.